build/configs/imxrt1050-evk/nxp_demo/defconfig

```diff
@@ -1115,7 +1116,24 @@ CONFIG_EXAMPLES_NXP_DEMO=y
 # CONFIG_EXAMPLES_SMART_TEST is not set
 # CONFIG_EXAMPLES_SPEECH_DETECTOR_TEST is not set
 # CONFIG_EXAMPLES_ST_THINGS is not set
-# CONFIG_EXAMPLES_TESTCASE is not set
+CONFIG_EXAMPLES_TESTCASE=y
+# CONFIG_DISABLE_MANUAL_TESTCASE is not set
+# CONFIG_EXAMPLES_TESTCASE_DRIVERS is not set
+CONFIG_EXAMPLES_TESTCASE_FILESYSTEM=y
```

Comment:
If this configuration is only for running test case, it might be better not to include it in this defconfig.

Reply:
Okay, I added a new defconfig 'imxrt1050-evk/tc'.